app/models/custom_attribute.rb

```diff
@@ -1,7 +1,19 @@
 class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
+  serialize :serialized_value
+  after_save do
+    update_column(:serialized_value, self.class.value_to_serialized_value(value)) if self.class.column_names.include?("serialized_value")
```

Comment:
Because im creating new custom_attributes in the test before migrating (https://github.com/ManageIQ/manageiq/pull/10896/files#diff-e06d01de8a97596a1715cdbd250ac2a1R6)